src/features/metadata-instance-editor/MetadataInstanceEditor.js

```diff
@@ -57,6 +62,8 @@ const MetadataInstanceEditor = ({
                     onModification={onModification}
                     onRemove={onRemove}
                     onSave={onSave}
+                    selectedTemplateKey={selectedTemplateKey}
+                    templateFilters={templateFilters ? convertTemplateFilters(templateFilters) : undefined}
```

Comment:
Is this needed? It looks like we're doing the same in MetadataSidebar.

Reply:
I added the ternary statement so that we would only pass the normalized `templateFilters` prop in the event that it is defined. I thought it would be cleaner to not pass `templateFilters` at all than to pass `templateFilters={null}`, but I can change this. What do you think?